x/posts/keeper/keeper.go

```diff
@@ -42,9 +42,26 @@ func NewKeeper(cdc *codec.Codec, storeKey sdk.StoreKey, paramSpace params.Subspa
 func (k Keeper) SavePost(ctx sdk.Context, post types.Post) {
 	store := ctx.KVStore(k.StoreKey)
 
+	// Retrieve the total number of posts, if null it will be equal to 0
+	var numberOfPosts sdk.Int
+	if bz := store.Get(types.PostTotalNumberPrefix); bz != nil {
+		k.Cdc.MustUnmarshalBinaryBare(bz, &numberOfPosts)
+	} else {
+		numberOfPosts = sdk.NewInt(0)
+	}
+
```

Comment:
I think that this can be simplified as 

```suggestion
	// Retrieve the total number of posts, if null it will be equal to 0
	var numberOfPosts = sdk.ZeroInt
	if store.Has(types.PostTotalNumberPrefix) {
		k.Cdc.MustUnmarshalBinaryBare(store.Get(types.PostTotalNumberPrefix), &numberOfPosts)
	}
```